### ISSUE TITLE
Understanding how multiple protocol IDs work

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#293]
I thought it would be useful to document the values of multiple protocol IDs.

When we decided to add multiple protocol IDs for a compliance app, I thought the value was recognizing an app could simultaneously satisfy more than one component type needed by a top level app. 

But I then got a bit confused when I read in commit 90930b00971e4628194a153076913123ad1ebe93, @JoshData states that "when a question specifies multiple as a list, the apps that can be started must implemented all of the IDs in the list (it's still possible to specify just one as a string)"

So I want to make sure I understand.  A component compliance app can list multiple protocols that the app satisfies and a top level compliance app (e.g., a compound or system compliance app) can also list in a question multiple protocol a component compliance app must satisfy to be used.

In short, a component app can satisfy multiple questions but most always satisfy all protocol conditions imposed by the question.

From the question perspective, any component compliance app that satisfies its listed conditions can be used.

The listed protocols in the compliance app is an `OR` condition while the listed protocols in the system/top level compliance app is an `AND` condition.

**Comment by gregelin** [comment]
This note needs to be added to the Apps.md page.

**Comment by gregelin** [comment]
OBE
